openrest4j-api/src/main/java/com/openrest/olo/dispatches/PickupDispatchInfo.java

```diff
@@ -1,6 +1,9 @@
 package com.openrest.olo.dispatches;

```

Comment:
I don't think we need to change the Info, see that the Address only exists in DeliveryDispatch